src/OnlineSales/Controllers/ContactsController.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace OnlineSales.Controllers;
 
-[Authorize]
+// [Authorize]
```

Comment:
you have commented Authorize attribute, is it should be commented?

Reply:
Sorry. Mistakenly committed. Its updated.